docs/_articles/en/soql/soql-builder.md

```diff
@@ -14,6 +14,8 @@ During beta, you can build simple query statements that include:
 - WHERE clause to filter your data
 - ORDER BY clause with support for ASC, DESC, NULLS FIRST, and NULLS LAST
 - LIMIT clause
+  <!-- ekapner, 2/19, add this to JA version -->
+  To dig deeper regarding SOQL syntax, or if you want to build more complex queries in the text editor, you can find all the details in the [SOQL and SOSL Reference guide](https://developer.salesforce.com/docs/atlas.en-us.soql_sosl.meta/soql_sosl/sforce_api_calls_soql_sosl_intro.htm).
```

Comment:
Small suggested edit (to shorten sentence a bit):  "To dig deeper into SOQL syntax or to build more complex queries in the text editor, see the [SOQL and SOSL Reference guide](https://developer.salesforce.com/docs/atlas.en-us.soql_sosl.meta/soql_sosl/sforce_api_calls_soql_sosl_intro.htm)."